src/test/java/com/google/api/codegen/testdata/python_doc_types_library.baseline

```diff
@@ -0,0 +1,65 @@
+============== file: google/cloud/proto/example/library_v1/types.py ==============
```

Comment:
I think using `organization_name` in that manner seems a bit magical.

I think a more ideal solution would be to have a property in the gapic yaml that holds the gapic package name and a property that holds the top-level package name. I think we should lean towards making these package names explicit rather than parsing the name out of an existing property.

ex:
```yaml
language_settings:
  python:
  package_name: google.cloud.ex_v1
  gapic_package_name: google.cloud.ex_v1.gapic
  proto_package_name: google.cloud.ex_v1.proto
```
Thoughts?